lib/twine/cli.rb

```diff
@@ -89,6 +89,12 @@ def self.parse(args)
           end
           options[:output_encoding] = e
         end
+        opts.on('-b', '--deliberate', 'Validate the strings file before formatting it') do
```

Comment:
Mmhh.. so only `--validate` without one dash/character short version? All characters in validate are already used except vor `-v` which is normally verbose.